tests/integration-test/src/tests/utils.rs

```diff
@@ -101,6 +101,20 @@ pub fn start_bpfd() -> Result<ChildGuard> {
     Ok(bpfd_process)
 }
 
+/// Retrieve bpfctl help test
+pub fn get_bpfctl_list_helptext() -> Result<String> {
+    let mut args = Vec::new();
+    args.push("list");
+    args.push("--help");
+    let output = Command::cargo_bin("bpfctl")?.args(args).ok();
+    let stdout = String::from_utf8(output.unwrap().stdout).unwrap();
+    let helptext = stdout.trim();
+    assert!(!helptext.is_empty());
```

Comment:
This works, but it's kind of changing the paradigm and doing the check in the util function instead of the test code.

Reply:
Moved out of the utils.rs file